### ISSUE TITLE
Pocket-Plus-Calibre-Plugin for Boox [PDF-based]

### ISSUE THREAD

**Issue by mmagnus** [mmagnus/Pocket-Plus-Calibre-Plugin#30]
I've recently moved from Kindle to Boox for my reading of Pocketed articles.

There is an easy way to get articles from Pocket to make a PDF and then transfer it to Boox.

```
cat pdf_TemplateScript.sh
ebook-convert Pocket.recipe .pdf #& open Pocket.mobi
mv -v Pocket.pdf "/Users/magnus/Dropbox/boox/Pocket `date +'%b %d %Y'`.pdf"
```

https://www.boox.com/noteair/

Mind, screenshots are in color ;-) but the e-ink display is not ;-)

![Screenshot_20210304-111505](https://user-images.githubusercontent.com/118740/109948949-1b24a300-7cdb-11eb-9818-146a448fdb52.png)
![Screenshot_20210304-111528](https://user-images.githubusercontent.com/118740/109948970-1f50c080-7cdb-11eb-8fb8-7988b5ad2e1d.png)
![Screenshot_20210304-111602](https://user-images.githubusercontent.com/118740/109948972-2081ed80-7cdb-11eb-91ec-b8dedea6ff0b.png)

and then sync back (from Boox to your computer) annotation for further processing:
![ 2021-03-04 at 13 22 32](https://user-images.githubusercontent.com/118740/109963489-c25e0600-7cec-11eb-85bf-d5f2c83b1316.jpg)

**Comment by alvaroreig** [comment]
hi there @mmagnus 

I've got a couple questions for you if you have the time.

- How happy are you with your Book, in general?
- If it has android and therefore you can access pocket directly, why generate the ebook in the first place?
- Do you use other android aps in it? In general, do they work ok? I mean, it's just a normal tablet except for the e-ink screen?

I ask you because I am considering buying one, but the main point would be reading articles in Pocket instead of generating and ebook. And using Todoist.

Thanks a lot, regards

**Comment by mmagnus** [comment]
Dear @alvaroreig I hope you're doing well in this pretty crazy world right now.

Great questions :-) and yeah, precisely to the point.

I was not sure about Boox at all. I love my Kindle, and with this plugin in here, I love my Kindle even more. But, at some point, pretty randomly, I bought a Boox device (Nova, 8'', a smaller one than my Boox Air), and I started to enjoy writing on it when I was reading. I use a lot pen and paper nowadays, and there is something about reading articles with the pen in hand. I really enjoy it.

POCKET
So here comes the big problem with Pocket app, it does not allow you to annotate your articles (write on them, you can highlight but this is no fun).  At least for me, it's a super fun way to read my Pocketed articles so I don't use the app on my Boox. (the app works fine, so this is pretty much personal preference). I also like to open a book and run article by article (very often skim many articles), and it's easier when you have a one opened file than go in Pocket from one article to another (small thing, but still makes a difference).

BOOX vs KINDLE
I decided to buy a device with a bigger screen for reading scientific papers. I tried many hacks how to read A4 scientific papers on Kindle, but this is usually a pain in ... Boox is great for that. But, at the same time I keep my Kindle because it is small and compact so it's great for regular e-books, where bigger screen is not needed and writing is not that essential.

APPS
Yeah, exactly, Boox is pretty much a tablet, so I installed there SSH server so I can sync my books over the Wifi. Other apps work as well, even Youtube ;-) but I don't put too many apps there to keep this distraction-free. Gmail works so that you can read e-mails; Dropbox works so that you can sync things as well, you can use Google Docs etc, this is pretty amazing. However, hand-writing in some of the apps might be too slow to be used. This happened to me with Evernote; somehow hand-writing was super slow in the apps and I could not use it.

SUMMARY
So to sum up, I really recommend Boox.  I'm very surprised that this is "Chinese" product but is of very high value. I usually use it for reading (less for hand-wiring notes) and annotating with pen for my scientific papers and Pocketed articles. I enjoy translate functionality and also speech-to-text. I've not used my Kindle for a few month right now ;-) Boox is my main device always on me.


![Screenshot_20200930-133516](https://user-images.githubusercontent.com/118740/110172551-98483900-7dfd-11eb-8eaa-e283f8f82d30.png)
![Screenshot_20200922-163236](https://user-images.githubusercontent.com/118740/110172571-9da58380-7dfd-11eb-8c06-36b1ebf694ea.png)

**Comment by alvaroreig** [comment]
I am ok, hope you are as well @mmagnus .
Thanks a lot for your input, I will keep that in mind. I am still debating which is the best use case for me right now, as I bounce from reading articles in Kindle to listening to them in Pocket.

**Comment by mmagnus** [comment]
Yeah, I was doing the same. BTW, you can listen to any text in Boox (which you can't do on Kindle), you can just select a text and let Boox read it for you. This is pretty amazing, I yet another reason I use Kindle less and less nowadays.

And often I want to archive long, important articles (sometimes behind paywall which often doesn't work with Pocket) as PDF to have a copy of that with annotation. So keeping all of these in the PDF reader on Boox is nice because you have everything in one place. In the same way, I print to PDFs mails that I want to read later.

So I have one folder to keep all things to read. 

But yet, these are personal preferences, and I guess everyone will prefer different configurations for themselves :)

**Comment by mmagnus** [comment]
Just some example of other apps, pretty slow and unusable 
https://youtu.be/reHTpNNltPo
